app/views/content/returning-to-teaching.md

```diff
@@ -42,6 +42,15 @@ calls_to_action:
       link_text: "Get a return to teaching adviser"
       link_target: "/tta-service"
       icon: "icon-person"
+
+  urgent-call-for-qualified-teachers:
+    name: simple
+    arguments:
+      text: "There's an urgent call for qualified teachers to get back into the classroom."
```

Comment:
```suggestion
      text: "There's an urgent call for qualified teachers to get back into the classroom. If you're eager to return to teaching, register to help."
```

Reply:
i prefer this edit, it tells people what they need to do!